iam_builder/templates.py

```diff
@@ -248,6 +260,28 @@ def get_s3_list_bucket_policy(list_of_buckets):
     }
     return policy
 
+def get_role_secrets(role):
+    if len(role)>1:
+        print("only one role permitted")
+        policy = {}
+    else:
+        secrets_path = f"arn:aws:ssm:*:*:parameter/alpha/airflow/{''.join(role)}/*"
+        policy = {
```

Comment:
This is a statement rather that a policy but I guess the name doesn't really matter